src/cocotb/triggers.py

```diff
@@ -996,48 +996,108 @@ def on_done(ret: Outcome[Any]) -> None:
 
 
 class ClockCycles(Waitable["ClockCycles"]):
-    r"""Fires after *num_cycles* transitions of *signal*.
-
-    If *rising* is ``True``, it fires after *num_cycle* transitions of *signal* from non-``1`` to ``1``.
-    If *rising* is ``False``, it fires after *num_cycle* transitions of *signal* from non-``0`` to ``0``.
+    r"""Finishes after *num_cycles* transitions of *signal*.
 
     :keyword:`await`\ ing this Trigger returns the ClockCycle object.
 
     Args:
         signal: The signal to monitor.
         num_cycles: The number of cycles to count.
-        rising: If ``True``, count rising edges; otherwise, count falling edges.
+        rising: If ``True``, count rising edges; if ``False``, count falling edges.
+        edge: Use the Edge Trigger type passed.
```

Comment:
```suggestion
        edge: Use the Edge trigger type passed.
```
?

Reply:
We do capitalize Task and Trigger and other keywords in other places. I will link against the edge-triggers here instead though.